### ISSUE TITLE
Security vulnerabilities found in jetty (dependency)

### ISSUE THREAD

**Issue by BrunoBonacci** [cognitect-labs/aws-api#173]
Hi,

`lein-nvd` is reporting a number of security vulnerabilities in jetty (one of transitive dependencies)

here the dep tree:
```
 [com.cognitect.aws/api "0.8.498"]
   [com.cognitect/http-client "0.1.105"]
     [org.eclipse.jetty/jetty-client "9.4.27.v20200227"]
       [org.eclipse.jetty/jetty-io "9.4.27.v20200227"]
     [org.eclipse.jetty/jetty-http "9.4.27.v20200227"]
     [org.eclipse.jetty/jetty-util "9.4.27.v20200227"]
```

here the vulnerabilities reported

```
+---------------------------------+------------------------------------------------+
| dependency                      | status                                         |
+---------------------------------+------------------------------------------------+
| jetty-util-9.4.27.v20200227.jar | CVE-2019-17638, CVE-2020-27218, CVE-2020-27216 |
+---------------------------------+------------------------------------------------+
```

here the CVE details
- [CVE-2019-17638](https://nvd.nist.gov/vuln/detail/CVE-2019-17638) - CRITICAL
- [CVE-2020-27216](https://nvd.nist.gov/vuln/detail/CVE-2020-27216) - HIGH
- [CVE-2020-27218](https://nvd.nist.gov/vuln/detail/CVE-2020-27218) - MEDIUM

All above vulnerabilities have been fixed jetty in the following version, please update to:

```
[org.eclipse.jetty/jetty-client "9.4.35.v20201120"] 
[org.eclipse.jetty/jetty-http   "9.4.35.v20201120"] 
[org.eclipse.jetty/jetty-util   "9.4.35.v20201120"] 
```

**Comment by dchelimsky** [comment]
Thanks for the report. I don't know how long it will be for us to get this through. In the mean time, for anybody unaware who may read this, you should be able to add these explicit jetty dependencies to your tools.deps, lein, or boot configuration to override the transitive dependencies.

**Comment by dchelimsky** [comment]
Fixed in 6222887df6e28e5cdee6cb0850bf7c5ab23b7194

Released in aws-api-0.8.505

Upgraded to cognitect/http-client-0.1.106, which depends on jetty 9.4.36.v20210114

**Comment by BrunoBonacci** [comment]
Hi,

thanks for the fix. 
Unfortunately, it appears that this version is now affected by another vulnerability.

```
  using nvd-clojure:  and dependency-check: 5.3.2
+-----------------------------------+----------------+
| dependency                        | status         |
+-----------------------------------+----------------+
| jetty-client-9.4.36.v20210114.jar | CVE-2020-27223 |
| jetty-http-9.4.36.v20210114.jar   | CVE-2020-27223 |
+-----------------------------------+----------------+

2 vulnerabilities detected. Severity: MEDIUM
```
